README.md

```diff
@@ -1,7 +1,7 @@
 # Battenberg
 
 [![image](https://img.shields.io/pypi/v/battenberg.svg)](https://pypi.python.org/pypi/battenberg)
-[![image](https://img.shields.io/travis/zillow/battenberg.svg)](https://travis-ci.org/zillow/battenberg)
+[![image](https://img.shields.io/travis/zillow/battenberg.svg)](https://github.com/zillow/battenberg/actions)
```

Comment:
Do we want to change or remove the badge? Guessing it's broken now 😅

Reply:
created an issue for this here: https://github.com/zillow/battenberg/issues/34

Probably change it out? But I'll leave that change for a future PR